alibi_detect/cd/keops/tests/test_learned_kernel_keops.py

```diff
@@ -0,0 +1,130 @@
+from itertools import product
+import numpy as np
+import pytest
+import torch
+import torch.nn as nn
+from typing import Callable, Optional, Union
+from alibi_detect.utils.frameworks import has_keops
+from alibi_detect.utils.pytorch import GaussianRBF as GaussianRBFTorch
+from alibi_detect.utils.pytorch import mmd2_from_kernel_matrix
+if has_keops:
+    from alibi_detect.cd.keops.learned_kernel import LearnedKernelDriftKeops
+    from alibi_detect.utils.keops import GaussianRBF
+    from pykeops.torch import LazyTensor
+
+n = 50
```

Comment:
Should add at least a comment (or a more descriptive variable name) to see at a glance what this is used for.